text/0002-preload-realm.md

```diff
@@ -0,0 +1,482 @@
+- Start Date: 2024-01-11
+- RFC PR: [electron/rfcs#4](https://github.com/electron/rfcs/pull/4)
```

Comment:
```suggestion
- RFC PR: [electron/rfcs#4](https://github.com/electron/rfcs/pull/4), [electron/rfcs#8](https://github.com/electron/rfcs/pull/8)
```

I think the filename also needs to be updated to `0008-preload-realm.md` to match the PR number.